proposals/distribution.md

```diff
@@ -142,8 +142,12 @@ The following entries should remain in the [scope table][scope] but not be addre
 * Allows listing of manifests: docker/distribution#2199
 
 [api.md]: https://github.com/docker/distribution/blob/5cb406d511b7b9163bff9b6439072e4892e5ae3b/docs/spec/api.md
+[catalog]: https://github.com/docker/distribution/blob/5cb406d511b7b9163bff9b6439072e4892e5ae3b/docs/spec/api.md#catalog
 [get-manifest]: https://github.com/docker/distribution/blob/5cb406d511b7b9163bff9b6439072e4892e5ae3b/docs/spec/api.md#pulling-an-image-manifest
 [iana-auth]: http://www.iana.org/assignments/http-authschemes/http-authschemes.xhtml
+[image-spec]: https://github.com/opencontainers/image-spec/
```

Comment:
quick question - some of these links are to versioned specs (the ones with the blob and commit) and others are to specific tags, and then others to the blame view. Is this intentional?

Reply:
> some of these links are to versioned specs (the ones with the blob and commit) and others are to specific tags, and then others to the blame view. Is this intentional?

Yeah.  This spec link floats, so distribution can add more endpoints if the imahe-spec gains more object types.  The Docker links have an explicit commit, because they haven't tagged a release since the last `api.md` comit (I think).  Otherwise I prefer pinning by tag.  Blames are for when I need a specific anchor that the rendered Markdown doesn't provide.

Reply:
ok cool! Just wanted to double check, makes perfect sense.